src/store/gameSlice.js

```diff
@@ -5,6 +5,7 @@ const initialState = {
   team1RemainingCards: 9,
   team2RemainingCards: 8,
   showResetButton: false,
+  guessesRemaining: 0,
```

Comment:
decided to move the guessesRemaining redux into gameslice since it's stored under game in firebase to be consistent.

i thought about manipulating the currentClue.number but didn't want to interfere with how were setting to  the game log so just decided to keep it separate for now.

Reply:
i think having this as a separate variable from currentClue.number is the right approach!
having guessesRemaining here is also fine i think!